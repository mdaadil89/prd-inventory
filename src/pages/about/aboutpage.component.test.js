import React from 'react'
import {mount, shallow} from 'enzyme'
import AboutPage from './aboutpage.component';
import toJson from 'enzyme-to-json';


describe('AboutPage Snapshot', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = mount(<AboutPage/>);
    }); 

    test('renders correctly', () => {
        expect(toJson(wrapper)).toMatchSnapshot(); 
    }); 
     
});

describe('About Page rendering of elements' , () => {
    let wrapper1;
    beforeEach(() => { 
        wrapper1 = shallow(<AboutPage/>)
    })
    
    it('renders one About React component', () => {
        expect(wrapper1.find('About').length).toEqual(1)
    })  
});