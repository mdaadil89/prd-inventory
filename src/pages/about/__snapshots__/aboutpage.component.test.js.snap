// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutPage Snapshot renders correctly 1`] = `
<AboutPage>
  <div>
    <About>
      <div>
        <h3>
          PRODUCT INVENTORY
        </h3>
        <p>
          This system is provisioned to view the list of products and allow only the registered and authenticated users to add, edit, delete the products and view the product detail. The user interface to the system has been implemented using React. The data about the products is managed in a JSON server. The client application interfaces with the JSON Server through http to retrieve and store data. This project was bootstrapped with Create React App.
        </p>
      </div>
    </About>
  </div>
</AboutPage>
`;
